src/lock_file/update.rs

```diff
@@ -113,6 +114,16 @@ impl<'p> LockFileDerivedData<'p> {
 
     /// Returns the up-to-date prefix for the given environment.
     pub async fn prefix(&mut self, environment: &Environment<'p>) -> miette::Result<Prefix> {
+        // Save an environment file to the environment directory
+        write_environment_file(
```

Comment:
Im fine with this, but I thought we talked about writing the file when:

* It doesnt exist yet.
* The prefix is actually updated.

Reply:
Lets keep it like this for now.